app/gui/src/controller/ide/desktop.rs

```diff
@@ -164,3 +161,27 @@ impl ManagingProjectAPI for Handle {
         .boxed_local()
     }
 }
+
+/// Select a new name for the project in a form of <suggested_name>_N, where N is a unique sequence
+/// number.
+fn choose_new_project_name(
+    mut existing_names: impl Iterator<Item = String>,
+    suggested_name: &str,
+) -> String {
```

Comment:
`ReferentName` should be used for project name.

Reply:
I think `ProjectName` suits the situation much better. I have also changed the Project Manager API definition to use `ProjectName` as well.

Reply:
To clarify, `ReferentName` seems to be `controller` entity, but here were are interacting with Project Manager, which apparently uses `ProjectName` (well, it should).